oracle/oracles/local_test.go

```diff
@@ -34,49 +34,52 @@ package oracles_test
 
 import (
 	"context"
+	"testing"
 	"time"
 
-	. "github.com/pingcap/check"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"github.com/tikv/client-go/v2/oracle"
 	"github.com/tikv/client-go/v2/oracle/oracles"
 )
 
-var _ = Suite(&testOraclesSuite{})
-
-type testOraclesSuite struct{}
-
-func (s *testOraclesSuite) TestLocalOracle(c *C) {
+func TestLocalOracle(t *testing.T) {
 	l := oracles.NewLocalOracle()
 	defer l.Close()
 	m := map[uint64]struct{}{}
 	for i := 0; i < 100000; i++ {
 		ts, err := l.GetTimestamp(context.Background(), &oracle.Option{})
-		c.Assert(err, IsNil)
+		require.Nil(t, err)
 		m[ts] = struct{}{}
 	}
 
-	c.Assert(len(m), Equals, 100000, Commentf("should generate same ts"))
+	assert.Len(t, m, 100000)
```

Comment:
it would better to keep comment (add the comment string as an extra argument)

Reply:
```suggestion
	assert.Len(t, m, 100000, "should generate same ts")
```

Reply:
Updated.